packages/@webex/media-helpers/src/sendslots-manager.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  SendSlot,
+  MediaType,
+  LocalStream,
+  MultistreamRoapMediaConnection,
+} from '@webex/internal-media-core';
+
+export default class SendSlotsManager {
+  private readonly slots: Map<MediaType, SendSlot> = new Map();
+  private readonly LoggerProxy: any;
+
+  constructor(LoggerProxy: any) {
+    this.LoggerProxy = LoggerProxy;
+  }
+
+  /**
+   * This method is used to create a sendSlot for the given mediaType and returns the created sendSlot
+   * @param {MultistreamRoapMediaConnection} mediaConnection MultistreamRoapMediaConnection for which a sendSlot needs to be created
+   * @param {MediaType} mediaType MediaType for which a sendSlot needs to be created (AUDIO_MAIN/VIDEO_MAIN/AUDIO_SLIDES/VIDEO_SLIDES)
+   * @param {boolean} active This is optional boolean to set the active state of the sendSlot. Default is true
+   * @returns {SendSlot} slot The created sendSlot
+   */
+  public createSlot(
```

Comment:
Wondering if we need a method called `getSlot` to get an existing slot, since this method throws an error if trying to create a send slot that already exists. Not sure if such a method would be useful.

Reply:
I agree. We need a `getSlot` method because we throw that error and what if we get a scenario in the future where we have to do a `createOrRetrieve`

Will add it.